src/libs/ReportUtils.ts

```diff
@@ -1452,30 +1523,24 @@ function requiresAttentionFromCurrentUser(option, parentReportAction = {}) {
 /**
  * Returns number of transactions that are nonReimbursable
  *
- * @param {Object|null} iouReportID
- * @returns {Number}
  */
-function hasNonReimbursableTransactions(iouReportID) {
+function hasNonReimbursableTransactions(iouReportID: string | undefined): boolean {
     const allTransactions = TransactionUtils.getAllReportTransactions(iouReportID);
-    return _.filter(allTransactions, (transaction) => transaction.reimbursable === false).length > 0;
+    return allTransactions.filter((transaction) => transaction.reimbursable === false).length > 0;
 }
 
-/**
- * @param {Object} report
- * @param {Object} allReportsDict
- * @returns {Number}
- */
-function getMoneyRequestReimbursableTotal(report, allReportsDict = null) {
-    const allAvailableReports = allReportsDict || allReports;
+function getMoneyRequestReimbursableTotal(report: OnyxEntry<Report>, allReportsDict?: OnyxCollection<Report>): number {
```

Comment:
You forgot to pass null here ig.

Reply:
I didnt forget, I think its not needed here `allReportsDict` is type of `OnyxCollection<Report>` which already means that it can be null also its an optional param so if we not pass this it will be undefined and below we have 
`const allAvailableReports = allReportsDict ?? allReports;` which means that if the allReportsDict will be undefined we will assign `allReports`

Reply:
I think we should keep this null.